apps/src/lib/kits/maker/ui/SetupGuide.jsx

```diff
@@ -75,13 +114,29 @@ export default class SetupGuide extends React.Component {
       );
     }
 
+    // Experiment 'microbit' renders additional description.
+    let isMicrobit = experiments.isEnabled('microbit');
+
     if (isCodeOrgBrowser() || isChromeOS() || isWebSerial) {
       return <SetupChecklist webSerialPort={webSerialPort} />;
     }
     return (
-      <Provider store={store}>
-        <Downloads />
-      </Provider>
+      <div>
+        <h1>{applabI18n.makerSetupPageTitle()}</h1>
+        <div style={isMicrobit ? style.boardDescriptions : {}}>
+          <Description
+            {...setupGuideContent.circuitPlayground}
+            divStyle={isMicrobit ? style.microbitDescription : {}}
```

Comment:
`microbitDescription` should probably be a more generic variable name since the styles can apply to the circuit playground description as well

Reply:
would something like `descriptionFlexCard` sound better?